.github/workflows/build-test-e2e.yml

```diff
@@ -21,9 +24,8 @@ jobs:
           path: '${{ env.BUILD_DIR }}/${{ env.BUILD_NAME }}'
           retention-days: 1
   playwright-tests:
-    if: ${{ github.ref_name == 'develop' || startsWith(github.ref_name, 'release-') || github.actor == github.repository_owner }}
     needs: [build]
-    runs-on: ${{ github.repository_owner == 'Sage-Bionetworks' && 'ubuntu-22.04-4core-16GBRAM-150GBSSD' || 'macos-latest' }}
+    runs-on: 'macos-latest'
```

Comment:
Always run playwright tests on a macos runner so environment is consistent between forks and in Sage repo

Reply:
Remind me, why do we only run on macos?

Reply:
We used the macOS runner because it's more powerful than the free Linux runner. 

@nickgros noticed that there are [usage limits](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits) on GitHub-hosted runners, which currently allow a maximum of 5 concurrent macOS jobs for a GitHub “Team” plan. Before switching over to the macOS runners, we would like to investigate the potential impact on the org's macOS runner resource availability -- tracking in SWC-6598.

For now, I'll change this PR to continue using the GitHub-hosted larger Linux runner.